cabot/metricsapp/api/elastic.py

```diff
@@ -1,11 +1,54 @@
 from elasticsearch import Elasticsearch
+from enum import Enum
+
+
+class SupportedMetrics(Enum):
+    # Response includes the field "value"
+    METRICS_SINGLE = ['min', 'max', 'avg', 'value_count', 'sum', 'cardinality', 'moving_avg', 'derivative']
+    # Response includes the field "values"
+    METRICS_MULTIPLE = ['percentiles']
 
 
 def create_es_client(urls, timeout):
     """
     Create an elasticsearch-py client
     :param urls: comma-separated string of urls
+    :param timeout: timeout for queries to the client
```

Comment:
Sure! what do you think the best default value is?

In my testing on stage I've had several timeouts with ES's default timeout of 10s (don't tell Libor), maybe 20-30? don't want to make it too high